src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -519,51 +522,46 @@ export class StudySummaryTab extends React.Component<
                     </div>
                 )}
 
-                {this.props.store.selectedSamples.result.length > 0 &&
-                    !this.store.loadingInitialDataForSummaryTab && (
-                        <div data-test="summary-tab-content">
-                            <div className={styles.studyViewFlexContainer}>
-                                {this.store.defaultVisibleAttributes
-                                    .isComplete && (
-                                    <ReactGridLayout
-                                        className="layout"
-                                        style={{
-                                            width: this.store.containerWidth,
-                                        }}
-                                        width={this.store.containerWidth}
-                                        cols={
-                                            this.store.studyViewPageLayoutProps
-                                                .cols
-                                        }
-                                        rowHeight={
-                                            this.store.studyViewPageLayoutProps
-                                                .grid.h
-                                        }
-                                        layout={
-                                            this.store.studyViewPageLayoutProps
-                                                .layout
-                                        }
-                                        margin={[
-                                            STUDY_VIEW_CONFIG.layout.gridMargin
-                                                .x,
-                                            STUDY_VIEW_CONFIG.layout.gridMargin
-                                                .y,
-                                        ]}
-                                        useCSSTransforms={false}
-                                        draggableHandle={`.${chartHeaderStyles.draggable}`}
-                                        onLayoutChange={
-                                            this.handlers.onLayoutChange
-                                        }
-                                        onResizeStop={this.onResize}
-                                    >
-                                        {this.store.visibleAttributes.map(
-                                            this.renderAttributeChart
-                                        )}
-                                    </ReactGridLayout>
-                                )}
-                            </div>
+                {!this.store.loadingInitialDataForSummaryTab && (
```

Comment:
the only change here is moving the `this.props.store.selectedSamples.result.length > 0` condition into `loadingInitialDataForSummaryTab`

Reply:
cool.  this definately seems to solve the problem.    is there a good reason you think we don't just test status of selectedSamples?   i.e. selectedSamples.isPending  ?